community_topics_workflow.md

```diff
@@ -8,64 +8,72 @@ This document describes the Ansible Community Topics workflow (hereinafter `Work
 - [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 21 days; usually it should not exceed 14 days, 21 days should only be used if it is known that a lot of interested persons will likely not have time to vote in a 14 days period).
-- [ ] The Committee person labels the topic with the `active-vote` label and moves the topic to the `Active Vote` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+- [ ] The Committee person labels the topic with the `active-vote` tag.
 - [ ] The Committee person adds `[Vote ends on $YYYY-MM-DD]` to the beginning of the topic's description.
+- A vote is actually two polls, one for the Steering Committee, one for everyone else. To create a vote in a topic:
+  - [ ] Create a new post in the topic.
+  - [ ] Click the `gear` button in the composer and select Build Poll.
+  - [ ] Click the `gear` in the Poll Builder for advanced mode.
+  - [ ] Set up the options (generally this will be Single Choice but other poll types can be used).
+  - [ ] Title it "Steering Committee vote" and "Limit voting" to the `Steering Committee`.
+  - [ ] Do not set the close date because this cannot be changed later.
+  - [ ] Results should be "Always Visible" unless there is some good reason for the SC votes not to be public.
+  - [ ] Submit the poll (the BBcode will appear in the post) and then repeat the above for the second poll.
+    - [ ] Title should be "Community vote".
+    - [ ] No group limitation.
 
 ### Voting result stage
 
 - [ ] The next day after the last day of the vote, the Committee person:
 
-  - [ ] Removes the `active-vote` label.
+  - [ ] Removes the `active-vote` tag.
   - [ ] Add a comment that the vote ended.
   - [ ] Changes the beginning of the topic's description to `[Vote ended]`.
-  - [ ] Counts the votes separating ones made by Community and made by Committee and creates a summary comment containing the result.
-  - [ ] Asks another Committee person to count the votes and provide the summary.
-- [ ] At least two Steering Committee members count the votes and summarize the result in comments.
+  - [ ] Creates a summary comment declaring the vote result.
 - [ ] The vote's result and the final decision are announced via the [Bullhorn](https://github.com/ansible/community/issues/546).
 
 ### Implementation stage
 
 - [ ] If the topic implies some actions (if it does not, just mark this as complete), the Committee person:
 
   - [ ] Assigns the topic to a person responsible for performing the actions.
-  - [ ] Add the `being_implemented` label to the topic.
+  - [ ] Add the `being_implemented` tag to the topic.
```

Comment:
Should this be changed to use a dash as well? Tags with underscores look strange to me.

Reply:
@felixfontein SGTM, changed, thanks, PTAL